OptimizelySDK/Entity/FeatureVariable.cs

```diff
@@ -35,6 +35,26 @@ public enum VariableType
         public string DefaultValue { get; set; }
         public VariableType Type { get; set; }
         public VariableStatus Status { get; set; }
-        
+
+        /// <summary>
+        /// Returns the feature variable api name based on VariableType.
+        /// </summary>
+        /// <returns>The feature variable type name.</returns>
+        /// <param name="variableType">Variable type.</param>
+        public static string GetFeatureVariableTypeName(VariableType variableType)
```

Comment:
Looks like this is just used for testing. Can we define it in the test file instead of making it a public property?

Reply:
No, it is used in GetFeatureVariableType<T> to get the method name from the ENUM value.